packages/amplication-client/package.json

```diff
@@ -17,9 +19,7 @@
     "events": "3.3.0",
     "formik": "2.2.9",
     "graphql": "15.0.0",
-    "lodash.debounce": "4.0.8",
-    "lodash.keyby": "4.6.0",
-    "lodash.omit": "4.5.0",
+    "lodash": "4.17.21",
```

Comment:
this change is not part of this PR. we actually added a separate PR to use a single import in all packages instead of separate imports. 
I think you may be right that on the client, we better use specific imports. 
If you think it is important, I suggest that you open a new issue to be fixed separately